### ISSUE TITLE
Python: Pool.get_label() returns True for bool labels

### ISSUE THREAD

**Issue by Vladimir-Kozlov** [catboost/catboost#2133]
Problem: in Python package, `Pool.get_label()` returns list of True if label passed is list of boolean
Code to reproduce:
```python
import catboost
pool = catboost.Pool([[0], [0]], label=[False, True])
assert pool.get_label() == [True, True]
pool = catboost.Pool([[0], [0]], label=[True, False])
assert pool.get_label() == [True, True]
pool = catboost.Pool([[0], [0]], label=[False, False])
assert pool.get_label() == [True, True]
```
catboost version: 1.0.6
Operating System: Ubuntu 20.04.04 LTS
CPU: Intel i7

**Comment by bgbg** [comment]
@andrey-khropov any news on this bug? Any ways to override it?

**Comment by Vladimir-Kozlov** [comment]
@bgbg This bug is still present as of 1.1.1. It can be avoided by converting labels to int.

**Comment by andrey-khropov** [comment]
Fixed in 103c724c8abc5fe4df6dfe7b3cdbb6e4d9bdd72d
